doc/user-guide/user-guide.rst

```diff
@@ -236,19 +236,23 @@ The steps in the build are as follows:
     - dimm
     - dmocps
     - dsm
+    - ess
     - genericcamera
+    - hvac
     - love-commander
     - love-csc
     - love-frontend
     - love-manager
     - love-manager-static
     - love-producer
+    - m1m3_sim
     - mthexapod
     - mtdome
     - mtdometrajectory
     - mtm2_sim
     - mtmount
     - mtrotator
+    - observatory-control
```

Comment:
Uhm.. this is not a component in the build step..

Reply:
Thanks for spotting that! Fixed.